google-ads-examples/src/main/java/com/google/ads/googleads/examples/extensions/RemoveEntireSitelinkCampaignExtensionSetting.java

```diff
@@ -0,0 +1,213 @@
+    private long customerId;
+
+    @Parameter(names = ArgumentNames.CAMPAIGN_ID, required = true)
+    private long campaignId;
+  }
+
+  public static void main(String[] args) {
+    RemoveEntireSitelinkCampaignExtensionSettingParams params =
+        new RemoveEntireSitelinkCampaignExtensionSettingParams();
+    if (!params.parseArguments(args)) {
+
+      // Either pass the required parameters for this example on the command line, or insert them
+      // into the code here. See the parameter class definition above for descriptions.
+      params.customerId = Long.parseLong("INSERT_CUSTOMER_ID_HERE");
+      params.campaignId = Long.parseLong("INSERT_CAMPAIGN_ID_HERE");
+    }
+
+    GoogleAdsClient googleAdsClient = null;
+    try {
+      googleAdsClient = GoogleAdsClient.newBuilder().fromPropertiesFile().build();
+    } catch (FileNotFoundException fnfe) {
+      System.err.printf(
+          "Failed to load GoogleAdsClient configuration from file. Exception: %s%n", fnfe);
+      System.exit(1);
+    } catch (IOException ioe) {
+      System.err.printf("Failed to create GoogleAdsClient. Exception: %s%n", ioe);
+      System.exit(1);
+    }
+
+    try {
+      new RemoveEntireSitelinkCampaignExtensionSetting()
+          .runExample(googleAdsClient, params.customerId, params.campaignId);
+    } catch (GoogleAdsException gae) {
+      // GoogleAdsException is the base class for most exceptions thrown by an API request.
+      // Instances of this exception have a message and a GoogleAdsFailure that contains a
+      // collection of GoogleAdsErrors that indicate the underlying causes of the
+      // GoogleAdsException.
+      gae.printStackTrace();
+      System.err.printf(
+          "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+          gae.getRequestId());
+      int i = 0;
+      for (GoogleAdsError googleAdsError : gae.getGoogleAdsFailure().getErrorsList()) {
+        System.err.printf("  Error %d: %s%n", i++, googleAdsError);
+      }
+      System.exit(1);
+    }
+  }
+
+  /** Runs the example. */
+  // [START RemoveEntireSitelinkCampaignExtensionSetting]
+  private void runExample(GoogleAdsClient googleAdsClient, long customerId, long campaignId) {
+    // Creates an operation to remove a campaign extension setting.
+    CampaignExtensionSettingOperation extensionSettingOperation =
+        CampaignExtensionSettingOperation.newBuilder()
+            .setRemove(
+                ResourceNames.campaignExtensionSetting(
+                    customerId, campaignId, ExtensionType.SITELINK))
+            .build();
+
+    // Retrieves all sitelink extension feed items for a customer and campaign.
+    List<String> extensionFeedItemResourceNames =
+        getSitelinkExtensionFeedItems(googleAdsClient, customerId, campaignId);
+
+    // Constructs operations to remove the extension feed item operations.
+    List<MutateOperation> extensionFeedItemOperations =
```

Comment:
Some parts that I found confusing is that the variables are not named close to their types.

For instance, `extensionSettingOperation` is kind of okay because it's a abbreviation of `CampaignExtensionSettingOperation`, but `extensionFeedItemOperations` here is quite confusing because it's not a list of `extensionFeedItemOperation`s, but rather a list of `MutateOperation`.

Could you rename this variable accordingly?